source/future.tex

```diff
@@ -674,34 +674,6 @@
 \tcode{current}.
 \end{itemdescr}
 
-\rSec1[depr.format]{Deprecated formatting}
-
-\rSec2[depr.format.syn]{Header \tcode{<format>} synopsis}
-
-\pnum
-The header \libheaderref{format}{format.syn} has the following additions:
```

Comment:
This reference was broken, I fixed it at head.

Reply:
I believe I have corrected this now, and force-pushed again.